src/main/java/ttt/GameEngine.java

```diff
@@ -50,16 +47,9 @@ public boolean isOver() {
         return board.isFinished();
     }
 
-    public String winner() {
-        return isWon() ? getWinner() : "Nothing";
-    }
-
-    public String getBoardStatus() {
-        return board.getStatus();
+    public Marks winningMark() {
+        if (!board.isWon()) return null;
```

Comment:
In Java, it is good to always use the brackets, even if there is only one line in the body. Without the brackets, the code becomes more error prone. Sometimes developers add another line to the body, but forget to add the brackets, so the new line ends up at a different scope than intended.